packages/react-components/src/trace-explorer/trace-explorer-time-range-data-widget.tsx

```diff
@@ -267,6 +267,23 @@ export class ReactTimeRangeDataWidget extends React.Component<
         }
     };
 
+    /**
+     * Allows public setting of active data and experiment data map.
+     * @param activeData
+     * @param map
+     */
+    public restoreState = (activeData: ExperimentTimeRangeData, map: Map<string, ExperimentTimeRangeData>): void => {
```

Comment:
maybe give call parameter `map` something more meaningful

Reply:
This has been changed to experimentDataMap